### PR TITLE
Jason/issue 1702

### LINKED ISSUE #1702: panning matched images results in region rendering out of sync
The following example is to generate a moment image from a cube then match the two images. If we pan on one image, the region on the other image is rendered out of sync, and vice versa. 

https://user-images.githubusercontent.com/20819712/142136506-30c8dec1-fda0-4ae9-aa7f-4af248f463ea.mov

this might be related to #1696

### PR DESCRIPTION
**Pull request by duidae**
closes #1702

### CODE REVIEW

**Comment by veggiesaurus** [comment]
The CI failures are not due to this branch, so ignore the failed checks

File: src/components/ImageView/RegionView/RegionViewComponent.tsx [hunk 1]
```diff
@@ -473,7 +473,7 @@ export class RegionViewComponent extends React.Component<RegionViewComponentProp
             stage.scale({x: refFrameZoom, y: refFrameZoom});
             const origin = scale2D({x: this.props.width / 2, y: this.props.height / 2}, 1 - refFrameZoom);
             const centerMovementCanvas = scale2D({x: refCenterMovement.x, y: -refCenterMovement.y}, refFrameZoom);
```
Comment (YuHsuan-Hwang) [inline src/components/ImageView/RegionView/RegionViewComponent.tsx]: We could combine the `scale2D` functions: `const centerMovementCanvas = scale2D({x: refCenterMovement.x, y: -refCenterMovement.y}, refFrameZoom / devicePixelRatio)`

Comment (duidae) [reply src/components/ImageView/RegionView/RegionViewComponent.tsx]: revised.

**Review by YuHsuan-Hwang — APPROVED** [review]

**Review by kswang1029 — APPROVED** [review]
tested with both non-retina and Retina display, issue is gone.

**Pull request merged**